CODEOWNERS

```diff
@@ -5,6 +5,7 @@ libs/ui/button @guilhermejcgois @oliveira-gust
 libs/ui/card-container @tigredonorte
 libs/ui/countdown-button @guilhermejcgois @oliveira-gust
 libs/ui/enhanced-title @tigredonorte
+libs/ui/gor-data-formatter @tigredonorte
```

Comment:
Note that all libs doesn't have prefix in folder and filename, mantain the pattern please

Reply:
ok